packages/react-components/react-tags/src/components/Tag/Tag.types.ts

```diff
@@ -1,20 +1,46 @@
+import { AvatarSize, AvatarShape } from '@fluentui/react-avatar';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
-import { Avatar } from '@fluentui/react-avatar';
+import { Button } from '@fluentui/react-button';
+
+export type TagContextValues = {
+  avatar: {
+    size?: AvatarSize;
+    shape?: AvatarShape;
+  };
+};
 
 export type TagSlots = {
   root: NonNullable<Slot<'div'>>;
-  content?: Slot<'span'>;
-  avatar?: Slot<typeof Avatar>;
-  icon?: Slot<'span'>;
-  primaryText?: Slot<'span'>;
-  secondaryText?: Slot<'span'>;
-  dismissButton?: NonNullable<Slot<'button'>>;
+
+  /**
+   * Slot for an icon or other visual element
+   */
+  media: Slot<'span'>;
+
+  /**
+   * A layout wrapper for the icon slot, the primaryText and secondaryText slots
+   */
+  content: Slot<'div'>;
+
+  icon: Slot<'span'>;
+
+  /**
+   * Main text for the Tag. Children of the root slot are automatically rendered here
+   */
+  primaryText: Slot<'span'>;
+
+  /**
+   * Secondary text that describes or complements the main text
+   */
+  secondaryText: Slot<'span'>;
```

Comment:
instead of `secondaryText` maybe you could use something like in table https://github.com/microsoft/fluentui/blob/e9c67ecb4f3b0ee1ee355e2c8eec17afa0b20a5a/packages/react-components/react-table/src/components/TableCellLayout/TableCellLayout.types.ts#L24-L27

Reply:
:D I was actually coping exactly this component -> how did you know 😱

Reply:
Learnt how to do pass children into primaryText(main) slot from it :D

Reply:
I kept the `primary text/secondary text` as in the previous PR. 
In design spec, I found there's both `primary content/secondary content` and `primary text/secondary text` :D Personally I prefer 'text' over 'content' as we have a 'content' slot, and it's kind of confusing to have another 'primary content'. But I'm open for changing this. I'll chat with the designer on the naming.